src/main/java/bio/terra/stairway/FlightDao.java

```diff
@@ -396,28 +415,44 @@ private void completeInner(FlightContext flightContext) throws SQLException {
     final String sqlDeleteFlightLog =
         "DELETE FROM " + FLIGHT_LOG_TABLE + " WHERE flightid = :flightId";
 
+    final String sqlDeleteFlightWorking =
+        "DELETE FROM " + FLIGHT_WORKING_TABLE + " WHERE flightid = :flightId";
+
     String serializedException =
         exceptionSerializer.serialize(flightContext.getResult().getException().orElse(null));
 
     try (Connection connection = dataSource.getConnection();
         NamedParameterPreparedStatement statement =
             new NamedParameterPreparedStatement(connection, sqlUpdateFlight);
-        NamedParameterPreparedStatement deleteStatement =
+        NamedParameterPreparedStatement deleteLogStatement =
+            new NamedParameterPreparedStatement(connection, sqlDeleteFlightLog);
+        NamedParameterPreparedStatement deleteWorkingStatement =
             new NamedParameterPreparedStatement(connection, sqlDeleteFlightLog)) {
 
       startTransaction(connection);
 
+      // TODO: Column output_parameters is being phased out in favor of table flightworking.
+      //       We continue to write the output map here temporarily for backward compatibility.
       statement.setString("outputParameters", flightContext.getWorkingMap().toJson());
+
+      statement.setInstant("completedTime", completedTime);
       statement.setString("status", flightContext.getFlightStatus().name());
       statement.setString("serializedException", serializedException);
       statement.setString("flightId", flightContext.getFlightId());
       statement.getPreparedStatement().executeUpdate();
 
       if (!keepFlightLog) {
```

Comment:
probably worth updating the documentation about this field to mention the working map table.

Reply:
Went with unique ID in flight log instead.